pkg/query/endpointset_test.go

```diff
@@ -269,13 +271,54 @@ func (e *testEndpoints) CloseOne(addr string) {
 	delete(e.srvs, addr)
 }
 
+func TestTruncateExtLabels(t *testing.T) {
+	testCases := []struct {
+		name            string
```

Comment:
Not useful name, we could just remove this field. Keeing `t.Run("", ...` name empty will generate nice number so all good.